File: Back/.Net/BetFriends.Application/Features/CreateBet/BetCreatedNotification.cs
using BetFriends.Domain.Bets;
using BetFriends.Domain.Members;
using MediatR;

namespace BetFriends.Application.Features.CreateBet;

public record BetCreatedNotification(BetId BetId, MemberId OwnerId, int Coins) : INotification;

File: Back/NodeJS/src/application/features/create-bet/BetCreatedNotification.ts
import { BetId } from "../../../domain/bets/BetId";
import { MemberId } from "../../../domain/members/MemberId";
import { INotification } from "../../Abstractions/Request/INotification";

export class BetCreatedNotification implements INotification {
    constructor(public BetId: BetId, public MemberId: MemberId, public Coins: number){}
    Name: string = BetCreatedNotification.name;
}